### PR TITLE
Remove bourbon dependency 

### PR DESCRIPTION
**Pull request by whmii**
https://github.com/whmii/cask/releases/edit/untagged-9328dda5fdb6e506d52f
https://github.com/whmii/cask/issues/4

### CODE REVIEW

File: app/assets/stylesheets/mixins/_grid-clearfix.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Provides an easy way to include a clearfix for containing floats.
+// Accepts options of `top`, `bottom` and `both`.
+
+@mixin grid-clearfix($location: bottom) {
```
Comment (tysongach) [inline app/assets/stylesheets/mixins/_grid-clearfix.scss]: This confusing me…the `$location` argument. I’m not sure I know why, when clearfixing, I would need to specify a location of something?

File: contrib/stylesheets/_example.scss [hunk 2]
```diff
@@ -179,11 +179,15 @@
...
+    @media #{$small} {
+      @include grid-column($columns: 2 of 5);
+    }
   }
 
   .col-3-of-5 {
-    @include grid-column($columns: 3 of 5);
+    @media #{$small} {
+      @include grid-column($columns: 3 of 5);
+    }
```
Comment (tysongach) [inline contrib/stylesheets/_example.scss]: Are the above changes a direct impact of removing Bourbon? If not, can these go into another PR?

File: app/assets/stylesheets/mixins/_grid-clearfix.scss (hunk 1 above)
Comment (whmii) [inline app/assets/stylesheets/mixins/_grid-clearfix.scss]: I'll change this to be more clear

File: contrib/stylesheets/_example.scss (hunk 2 above)
Comment (whmii) [inline contrib/stylesheets/_example.scss]: I'll refactor this in to another pr
